config/nginx.sample.conf

```diff
@@ -1,7 +1,6 @@
 # Additional MIME types that you'd like nginx to handle go in here
 types {
     text/csv csv;
-    application/wasm wasm;
```

Comment:
Will you be able to link to where else this type has been specified?

Reply:
Here: https://github.com/discourse/discourse/blob/9db139ecb1d01ceec174e2449b06f5a26f8430a9/config/nginx.sample.conf#L49

Reply:
I'm not sure I follow here because the `gzip_types` directive is used to specify which MIME types are supposed to be gzipped. Are you able to post the full warning message that you see in nginx here?

Reply:
I get a total of 2 warnings, so I created 2 PR's. As soon as I adjust the lines the warnings are gone and I don't see any errors in the forum, since the pr my forum ran with the changes.

```
nginx -t
nginx: [warn] duplicate extension "wasm", content type: "application/wasm", previous content type: "application/wasm" in /etc/nginx/conf.d/discourse.conf:4
nginx: [warn] the "listen ... http2" directive is deprecated, use the "http2" directive instead in /etc/nginx/conf.d/discourse.conf:60
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful
```